src/main/scala/mesosphere/marathon/MarathonSchedulerActor.scala

```diff
@@ -424,8 +424,13 @@ class SchedulerActions(
               else instanceTracker.setGoal(i.instanceId, Goal.Decommissioned)
             }
 
+            logger.info("waiting for the changeGoalsFuture")
```

Comment:
This seems like a debug log.

Reply:
Oh, yeah, forgot about that! Good catch!